includes/Checker/Checks/I18n_Usage_Check.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Translation check class.
```

Comment:
```suggestion
 * Class WordPress\Plugin_Check\Checker\Checks\I18n_Usage_Check
```

For consistency, use a similar description.

Reply:
@mukeshpanchal27 Updated now, thank you!